test/integration/gateways/monei_test.exs

```diff
@@ -62,4 +62,5 @@ defmodule Gringotts.Integration.Gateways.MoneiTest do
     assert config[:adapter] == Gringotts.Gateways.Monei
   end
 
+  defp amount(amount, currency \\ "EUR"), do: %{amount: Decimal.new(amount), currency: currency}
```

Comment:
same here as above comment, modify test cases to show it works with dependencies? what do you think?

Actually compatibility with external libs can be tested in a single file I am thinking.

Thoughts ? @kartikjagdale @arjun289 @oyeb

Reply:
Not without adding them as dependencies that are available in `test` environment.

Reply:
So do we agree to add `ex_money` and `monteized` to our dependencies?

Reply:
Yes, also keep the `any` fallback just in case. We need more documentation stating out of box support for these libs. @oyeb